src/balanceTransfer.ts

```diff
@@ -1,81 +1,31 @@
 import Metadata from '@polkadot/metadata';
 import { createType, TypeRegistry } from '@polkadot/types';
-import { SignerPayloadJSON } from '@polkadot/types/types';
 
 import { EXTRINSIC_VERSION, ONE_SECOND } from './util/constants';
+import { BaseTxInfo, UnsignedTransaction } from './util/interfaces';
 
-/**
- * JSON format for an unsigned transaction
- */
-export interface UnsignedTransaction extends SignerPayloadJSON {
-  /**
-   * The SCALE-encoded metadata, as a hex string. Can be retrieved via the RPC
-   * call `state_getMetadata`
-   */
-  metadataRpc: string;
-}
-
-export interface TxInfo {
+export interface TxInfoTransfer extends BaseTxInfo {
```

Comment:
Following the `staking` folder, I would also in this case create a folder called `balances`, with 2 files `transfer.ts` and `transferKeepAlive.ts`.

Can be done in a future PR, to keep this one only about staking.